test/test_distributions.py

```diff
@@ -2623,6 +2623,40 @@ def test_kl_monte_carlo(self):
                 'Actual (analytic): {}'.format(actual),
             ]))
 
+    # Multivariate normal has a separate Monte Carlo based test due to the requirement of random generation of
+    # positive (semi) definite matrices. n is set to 5, but can be increased during testing.
+    def test_kl_multivariate_normal(self):
+        set_rng_seed(0)  # see Note [Randomized statistical tests]
+        n = 5  # Number of tests for multivariate_normal
+        print('Testing KL(MultivariateNormal, MultivariateNormal) using Monte Carlo')
+        for i in range(0, n):
+            loc = []
+            psd_cov = []
```

Comment:
I guess a simple alternative would be to use `transform_to(constraints.lower_cholesky)`, e.g.
```py
loc = torch.randn(4, 4)
scale_tril = transform_to(constraints.lower_cholesky)(torch.randn(4, 4))
p = MultivariateNormal(loc, scale_tril=scale_tril)
```

Reply:
This seems tidy. Thank you.